docs/cloud/connecting-services.mdx

```diff
@@ -5,45 +5,298 @@ icon: "link"
+import dev.restate.sdk.auth.signing.RestateRequestIdentityVerifier
+import dev.restate.sdk.http.vertx.RestateHttpServer
+import dev.restate.sdk.kotlin.endpoint.endpoint
+
+fun main() {
+  RestateHttpServer.listen(
+      endpoint {
+        bind(MyService())
+        requestIdentityVerifier =
+            RestateRequestIdentityVerifier.fromKeys(
+                "publickeyv1_w7YHemBctH5Ck2nQRQ47iBBqhNHy4FV7t2Usbye2A6f",
+            )
+      })
+}
+```
+
+
+```python Python {"CODE_LOAD::python/src/develop/serving.py#identity"} 
+app = restate.app(
+    services=[my_service],
+    identity_keys=["publickeyv1_w7YHemBctH5Ck2nQRQ47iBBqhNHy4FV7t2Usbye2A6f"],
+)
+```
+
+
+```go Go {"CODE_LOAD::go/develop/serving.go#identity"} 
+if err := server.NewRestate().
+  Bind(restate.Reflect(MyService{})).
+  WithIdentityV1("publickeyv1_w7YHemBctH5Ck2nQRQ47iBBqhNHy4FV7t2Usbye2A6f").
+  Start(context.Background(), ":9080"); err != nil {
+  log.Fatal(err)
+}
 ```
 
-### 3. Run the tunnel
+
+</CodeGroup>
+
+
+You can find more information on how to deploy Restate to various serverless platforms in the following guides:
+- [Vercel](https://docs.restate.dev/services/deploy/vercel)
+- [Cloudflare Workers](https://docs.restate.dev/services/deploy/cloudflare-workers)
+- [Deno Deploy](/services/deploy/deno-deploy)
+
+Note that, if you are using public endpoints, HTTPS must be used between Restate and your services. You can terminate TLS at the service endpoint directly, but it's likely easier to use a fronting load balancer like an AWS NLB.
+
+## AWS Lambda functions
+
+Restate Cloud can securely invoke your AWS Lambda functions by assuming an AWS identity that has the right permissions.
+
+### Setting up the invoker role
+
+Restate Cloud must assume an AWS identity with the permission to perform `lambda:InvokeFunction`, within the same AWS account where the Lambda function is deployed.
+
+Note that the Restate Cloud role is separate from your Lambda function’s execution role.
+The execution role is used by your function to perform its tasks, while the invoker role grants Restate Cloud permission to invoke your service handler functions — and nothing else.
+
+You can set up the role either by creating an IAM role yourself or by using the Restate CDK construct library.
+
+#### IAM trust policy for Restate Cloud
+To create a new IAM role, you can copy the trust policy directly from the Restate Cloud UI, by visiting [Developers > Security > AWS Lambda](https://cloud.restate.dev/to/developers/integration#lambda).
+This trust policy allows the Restate Cloud principal (e.g. `us.restate.cloud`) to assume the role, but only on behalf of the specified environment ID.
+
+#### Using AWS CDK
+
+When you use the [Restate CDK construct library](https://github.com/restatedev/cdk) to deploy
+Lambda handlers, the provided invoker role will automatically be granted access
+to invoke the corresponding functions.
```

Comment:
Maybe the npmjs page is a more appropriate place to direct people? Your call, both places show the README quite prominently.

```suggestion
You can use AWS CDK together with the [Restate CDK construct library](https://www.npmjs.com/package/@restatedev/restate-cdk) to deploy
Lambda handlers, manage the invoker role, and automatically register Lambda-backed services with a Restate (Cloud) environment.
```